File: src/Example3Java/EX02.java
package Example3Java;

import java.util.Scanner;

public class EX02 {
    public static void main(String[] args) {
        System.out.print("Enter number : ");
        String number = new Scanner(System.in).nextLine();
        String phone1 = number.substring(0, 3);
        String phont2 = number.substring(4);
        System.out.println("Phone number : " + phone1 + "-" + phont2);
    }
}

File: src/Example3Kotlin/EX02.kt
package Example3Kotlin

import java.util.Scanner

object EX02 {
    @JvmStatic fun main(args: Array<String>) {
        print("Enter number : ")
        val number = Scanner(System.`in`).nextLine()
        println("Phone number : ${number.substring(0, 3)}-${number.substring(4)}")
    }
}
